README.md

```diff
@@ -108,7 +108,13 @@
  | 53         | sgx\_ecdsa                 | sgx\_ecdsa\_qve            |
 
  For instance priority, the higher, the stronger. By default, Librats will select the **highest priority** instance to use.
+ 
+ ## Run cert-app
 
+ ```
+ cd /usr/share/librats/samples
+ ./cert-app
+ ```
```

Comment:
What about put those in [samples/cert-app/README.md](https://github.com/inclavare-containers/librats/blob/d841d2b27f374371e031467707d4b40c75184696/samples/cert-app/README.md)?

Reply:
OK, I've migrated the [Run cert-app] and [Specify the instance type] sections into samples/cert-app/README.md